src/components/checkbox/checkbox.spec.js

```diff
@@ -246,5 +247,45 @@ describe('mdCheckbox', function() {
       expect(isChecked(checkbox)).toBe(false);
       expect(checkbox.hasClass('ng-invalid')).toBe(true);
     });
+
+    describe('with the md-indeterminant attribute', function() {
+
+      it('should set md-indeterminant attr to true by default', function() {
+        var checkbox = compileAndLink('<md-checkbox md-indeterminant></md-checkbox>');
+
+        expect(checkbox).toHaveClass(INDETERMINATE_CSS);
+      });
+
+      it('should set its HTMLInputElement.indeterminate to be undefined when not md-indeterminant', function() {
```

Comment:
I would rather say: `should not set the indeterminate property when md-indeterminate is false.`